packages/nsq/nsq.0.5.2/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Ryan Slade <ryanslade@gmail.com>"
+authors: "Ryan Slade <ryanslade@gmail.com>"
+homepage: "https://github.com/ryanslade/nsq-ocaml"
+bug-reports: "https://github.com/ryanslade/nsq-ocaml/issues"
+dev-repo: "git+https://github.com/ryanslade/nsq-ocaml.git"
+build: [["dune" "build" "-p" name "-j" jobs]]
+synopsis: "A client library for the NSQ messaging platform"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "2.0"}
```
According to the dune-project file, you require version 2.0 at least.
The `build` tag cannot be used with dune due to forward-dependencies issues.

Reply:
Can you please upstream this change?

Reply:
Will do, thanks